samples/other/streaming/src/main/java/org/eclipse/dataspaceconnector/transfer/demo/protocols/common/AbstractQueuedProvisioner.java

```diff
@@ -32,7 +32,7 @@
 
     private final AtomicBoolean active = new AtomicBoolean();
 
-    private long provisionWait = 100;
+    private long provisionWait = 5000;
```

Comment:
this would be a candidate for the "Further Notes" section in the PR description explaining why this timeout was increased.